specification/hybridkubernetes/resource-manager/Microsoft.Kubernetes/stable/2021-10-01/examples/ConnectedClustersListClusterCredentialResult.json

```diff
@@ -0,0 +1,24 @@
+{
+  "parameters": {
+    "subscriptionId": "1bfbb5d0-917e-4346-9026-1d3b344417f5",
+    "resourceGroupName": "k8sc-rg",
+    "clusterName": "testCluster",
+    "api-version": "2021-10-01",
+    "Properties": {
+      "authenticationMethod": "AAD",
+      "clientProxy": true
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "kubeconfigs": [
```

Comment:
Should this have hybrid config in response?

Reply:
Added it